test/collection_functions.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require '../collection_functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Collection_main extends TestCase
+{
+
+    public function test_feedback_malformed()
```

Comment:
rename to test print results malformed or something like that

Reply:
renamed